xml/ha_storage_protection.xml

```diff
@@ -467,347 +617,254 @@ Timeout (allocate) : 2
-<!-- bnc#891346 -->
-       Which value to set for <literal>stonith-timeout</literal> depends on
-       the <literal>msgwait</literal> timeout. The
-       <literal>msgwait</literal> timeout should be longer than the maximum
-       allowed timeout for the underlying IO system. For example, this is 30
-       seconds for plain SCSI disks. Provided you set the
-       <literal>msgwait</literal> timeout value to 30 seconds, setting
-       <literal>stonith-timeout</literal> to 40 seconds is appropriate.
-      </para>
-<!--Use the following formula to set the minimum value of
-        <literal>stonith-timeout</literal>:</para>
-       <screen>stonith-timeout >= 1.2 * msgwait</screen>-->
-      <para>
-       Since node slots are allocated automatically, no manual host list
-       needs to be defined. For more details, refer to <link xlink:href="https://www.suse.com/support/kb/doc.php?id=7016305"/>.
-      </para>
-     </step>
-     <step>
-      <para>
-       Disable any other fencing devices you might have configured before,
-       since the SBD mechanism is used for this function now.
-      </para>
-     </step>
-    </procedure>
+   </step>
+   <step>
     <para>
-     After the resource has started, your cluster is successfully configured
-     for shared-storage fencing and will use this method in case a node
-     needs to be fenced.
+     Run <command>crm</command> <option>configure</option>.
     </para>
-   </sect3>
-   <sect3 xml:id="sec.ha..storageprotection.sgpersist">
-    <title>Configuring an sg_persist Resource</title>
-    <note>
-     <title>SCSI Disk Compatibility</title>
-     <para>
-      Any backing disks for <literal>sg_persist</literal> must be SCSI disk
-      compatible. <literal>sg_persist</literal> only works for devices like SCSI
-      disks or iSCSI LUNs. Do <emphasis>not</emphasis> use it for IDE, SATA,
-      or any block devices which do not support the SCSI protocol.
-     </para>
-    </note>
-    <remark>toms 2014-09-10: FATE#312345</remark>
-    <procedure>
-     <step>
-      <para>
-       Log in as &rootuser; and start a shell.
-      </para>
-     </step>
-     <step>
-      <para>
-       Run the following commands to create the primitive resources
-       <literal>sg_persist</literal>:
-      </para>
-<screen>&prompt.root;<command>crm</command> configure
+   </step>
+   <step>
+    <para>Enter the following:</para>
+    <screen>
+&prompt.crm.conf;<command>property</command> stonith-enabled="true" <co xml:id="co.ha.sbd.st.enabled"/>
+&prompt.crm.conf;<command>property</command> stonith-timeout="40s" <co xml:id="co.ha.sbd.st.timeout"/>
+&prompt.crm.conf;<command>primitive</command> stonith_sbd stonith:external/sbd <co xml:id="co.ha.sbd.st.rsc"/>\
+  params pcmk_delay_max=30 <co xml:id="co.ha.sbd.pm_delay_max"/></screen>
+    <calloutlist>
+     <callout arearefs="co.ha.sbd.st.enabled">
+      <para>
+       This is the default configuration, because clusters with &stonith; are not supported.
```

Comment:
s/with/without/

Reply:
Argh, very good catch, thanks! Fixed in both places.